test/test_helpers/sphinx.rb

```diff
@@ -12,13 +12,12 @@ def real_time_run
         init
         start index: false
 
-        disabled = Mocha::Mockery.instance.stubba.stubba_methods.any? {|m| m.stubbee == SphinxIndexationWorker && m.method_name == :perform}
-        enable_search_jobs! if disabled
+        enable_search_jobs!
```

Comment:
I think it needs to check whether it was disabled before. It seems to me `enabled` is the default state. Isn't it?

Reply:
We have 
```
module TestHelpers
  module Sphinx
    def self.included(base)
      base.setup(:disable_search_jobs!)
    end
...
```